sdk/service/service.go

```diff
@@ -0,0 +1,89 @@
+package servicesdk
+
+import (
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	"github.com/cskr/pubsub"
+	"github.com/docker/docker/pkg/archive"
+	"github.com/mesg-foundation/core/container"
+	"github.com/mesg-foundation/core/database"
+	"github.com/mesg-foundation/core/service"
+	"github.com/mesg-foundation/core/service/manager"
+	"github.com/mesg-foundation/core/utils/dirhash"
+	"github.com/mr-tron/base58"
+	uuid "github.com/satori/go.uuid"
+)
+
+// ServiceSDK exposes service APIs of MESG.
+type ServiceSDK struct {
```

Comment:
What about just naming it `Service`? the filepath already give the context of sdk.

Reply:
or `SDK`?

Reply:
No, it's only about the service

Reply:
done